### ISSUE TITLE
Training and validation image is of different size?

### ISSUE THREAD

**Issue by timothylimyl** [VainF/DeepLabV3Plus-Pytorch#53]
Hi,

https://github.com/VainF/DeepLabV3Plus-Pytorch/blob/master/main.py#L129

When the dataset is Cityscapes, the training size is set by `opts.crop_size` while the validation set uses the original resolution. How does this works?
